docs/population.md

```diff
@@ -354,6 +364,19 @@ To make use of the urban type, the following data is needed:
 
 Then, you should be able to run the pipeline with the configuration explained above.
 
+### Filter household travel survey data
+
+The pipeine filter out by default observations, from household travel survey used, which live or work outside of the area used in the simulation.
+Unfortunately, household travel survey data is not always well distributed at scales other than national or regional and may be incomplete for area used in the simulation.
```

Comment:
However, the national HTS (ENTD and EMP) may be very sparse in rural and undersampled areas.